heuristics/hyades_heuristic.py

```diff
@@ -0,0 +1,70 @@
+from cfmm_base import infotodict as cfmminfodict
+from cfmm_base import create_key
+
+def infotodict(seqinfo):
+    """Heuristic evaluator for determining which runs belong where
+    allowed template fields - follow python string module:
+    item: index within category
+    subject: participant id
+    seqitem: run number during scanning
+    subindex: sub index within group
+    """
+
+    # call cfmm for general labelling and get dictionary
+    info = cfmminfodict(seqinfo)
+
+    # key definitions
+    # anat  (already included in the cfmminfodict)
+    #t1w = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_run-{item:02d}_T1w')
+    #flair  = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_run-{item:02d}_FLAIR')
+    #t2w = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_run-{item:02d}_T2w')
+    
+    # dwi. added _acq-mde as a key to identify this as a MDE scan
+    dwi_mde = create_key('{bids_subject_session_dir}/dwi/{bids_subject_session_prefix}_acq-mde_dir-AP_run-{item:02d}_dwi')
+    # Save the RPE (reverse phase-encode) B0 image as a fieldmap (fmap).
+    fmap_rev_phase =  create_key('{bids_subject_session_dir}/fmap/{bids_subject_session_prefix}_dir-PA_run-{item:02d}_epi')
+    
+    # MTS collection
+    mt_off = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_flip-1_mt-off_run-{item:02d}_MTS')
+    mt_on  = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_flip-1_mt-on_run-{item:02d}_MTS')
+    mt_t1w  = create_key('{bids_subject_session_dir}/anat/{bids_subject_session_prefix}_flip-2_mt-off_run-{item:02d}_MTS')
+    
+    # Fill data dictionary
+    #info[t1w]=[]
+    #info[flair]=[]
+    #info[t2w]=[]
+    info[dwi_mde]=[]
+    info[fmap_rev_phase]=[]
+    info[mt_off]=[]
+    info[mt_on]=[]
+    info[mt_t1w]=[]
+    
+
+    # Criteria
+    for idx, s in enumerate(seqinfo):
+        # anat  (already included in the cfmminfodict)
+        #if ('T1w_MPR**' in s.series_description):
+        #    info[t1w].append(s.series_id)
+        #if ('t2_space_dark-fluid_sag_p2_iso' in s.protocol_name):
+        #    info[flair].append(s.series_id)
+        #if ('T2w_SPC' in s.protocol_name):
+        #    info[t2w].append(s.series_id)
+
+        #  dwi_mde
+        if ('UFA_AP' in s.protocol_name):
+            info[dwi_mde].append(s.series_id)
```

Comment:
To substitute the series id for the run, you'll want to do something similar to:

`info[dwi_mde].append({'item': s.series_id})`

You'll want to do this for all of the scans in the heuristic that contains the run entity.

Reply:
Ok, thanks for the comment.